src/inverseRobot.js

```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const arr1 = Object.keys(robot);
+  const arr2 = Object.values(robot);
```

Comment:
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)